src/main/java/core/basesyntax/parsers/ParseFromFile.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.parsers;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseFromFile {
```

Comment:
class names should be nouns

also, do you use openscv library that you added in pom.xml?

Reply:
actually, I ended without using it(  probably don't need it at all